docs/README.md

```diff
@@ -740,6 +741,19 @@ With proper concurrency logic in place, the latest run might have been cancelled
           workflow: my-workflow.yml
 ```
 
+### github-dependabot-check
+
+Action to check if a pull request is opened by dependabot with a specific label.
+
+```yaml
+      - name: dependabot check
+        id: dependabot
+        uses: ./.github/actions/github-dependabot-check
+        with:
+          gh-token: ${{ secrets.GITHUB_TOKEN }}
+          label: github_actions
+```
```

Comment:
would be cool to have a more comprehensive/real world example because right now I am struggling to see how much reusable this is (how do you actually use the result of this action?)

Reply:
https://github.com/Activiti/Activiti/pull/5020/files#diff-1578e1a2c827dc71148f1362dddb7cab08c67fe01518fb0f198fc3b571e6cba7R30

@gionn  this is how we are going to use it.. basically we need an action to identify if a PR was opened by dependabot with a specific label gh_action to avoid the deploy or any other CI operation not needed to save time

Reply:
@gionn  I've added a more real use case. let me know

Reply:
the only reason why I have been asked to move the action in this public is to reuse it across the 8 or more BE automate repos. If the action is only returning the metadata I don't see the point to have an action I can do the call in each repo instead of referencing the action.

Reply:
What if Dependabot PRs aren't the only ones you want to skip? Other automations might use github-action[bot] or a custom GitHub user. And what if you're merging a user-created PR that shouldn't trigger propagation because the changes don't need it?
What if you have a complex workflow for which you could have a different outcome than just skip the job?

Same desired behavior, but very different triggering conditions.

Reply:
right now this is a very specific scenario we would like to tackle. 
dependabot + gh_action 
once we will collect different data and cases we can change/improve the action. 
Again, initially I was creating the action as part of the repo but I was told to export it as shared since we have several BE repo that are suffering from this. 
On FE we use the Nx tool that allows us to handle different scenarios.

Reply:
Indeed we would like to tackle this and see if it really helps us with unneeded propagations. 
I like what you're suggesting @gionn and I agree it would be much more generic, but I don't want us to have to do hardcode that user is dependabot, and we can use this action several times if we want to check more than one label (and do the AND/OR custom logic) on the caller side.
If we made the name more generic, and introduced an "actor" input that would default to `dependabot[bot]`, do you think that could be enough as a first step?

Reply:
Reading the README, I even wonder if it would make sense to have github_actions label by default too.

Also our alternative is pushing to Activiti/Activiti actions for easier reuse everywhere if that's too specific (I did that once for [a similar need](https://github.com/Activiti/Activiti/pull/4395)), we can go back to that even if it's not the cleanest for our version references

Reply:
here is a suggestion (better name for the action to be found): https://github.com/Alfresco/alfresco-build-tools/pull/1015/commits/c290b059231e3003a52c9130b4c8904eb12e8f7e

Reply:
my understanding is that the action logic makes it work both on push (after PR merge) and on pull_request, right @mauriziovitale? even if we'll use it on push only for our use case - I have other use cases I see now (to detect propagation PRs for instance) where this could be reused

Reply:
While it can technically be used on `pull_request` I would probably discourage using it? In case of a `pull_request` event I'd assume we'd already have both the author and the labels as part of the event and it's fairly trivial to extract it & run conditional logic based on it without having to make a gh api call 🤔

Reply:
you should always fetch the label from api if you want to get the latest status and not only rely on those originally added when the PR was created. Btw I am fine with any solution.

Reply:
Yes as Maurizio says we've been fetching labels from API anyway as it's more reliable

`github-pr-check`?

Reply:
`github-pr-check-metadata` sounds good to me

Reply:
Rename done